server/events/project_command_builder.go

```diff
@@ -332,6 +342,15 @@ func (p *DefaultProjectCommandBuilder) buildPlanAllCommands(ctx *command.Context
 		ctx.Log.Debug("moduleInfo for %s (matching %q) = %v", repoDir, p.AutoDetectModuleFiles, moduleInfo)
 		modifiedProjects := p.ProjectFinder.DetermineProjects(ctx.Log, modifiedFiles, ctx.Pull.BaseRepo.FullName, repoDir, p.AutoplanFileList, moduleInfo)
 		ctx.Log.Info("automatically determined that there were %d projects modified in this pull request: %s", len(modifiedProjects), modifiedProjects)
+
+		if p.EnableRegExpCmd && cmd.RepoRelDir != "" {
```

Comment:
This code looks to be the same as lines 303 to 309. 

Can we encapsulate this into a function to keep the code as DRY as possible?

Reply:
Or is there a better way to structure this so we can run this block for both if and else blocks?